storage/backup.py

```diff
@@ -75,61 +75,51 @@
     * :red:`timer-01sec`
 """
 
-#------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
 from __future__ import absolute_import
+
 from io import BytesIO
 
-#------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
 _Debug = False
 _DebugLevel = 12
 
-#------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
+import gc
 import os
 import sys
 import time
-import gc
 
 try:
     from twisted.internet import reactor  # @UnresolvedImport
 except:
-    sys.exit('Error initializing twisted.internet.reactor in backup.py')
+    sys.exit("Error initializing twisted.internet.reactor in backup.py")
 
-from twisted.internet.defer import maybeDeferred, Deferred, succeed
+from twisted.internet.defer import Deferred, maybeDeferred, succeed
 
-#------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
 
 if __name__ == "__main__":
     import os.path as _p
-    sys.path.append(_p.join(_p.dirname(_p.abspath(sys.argv[0])), '..'))
-
-#------------------------------------------------------------------------------
 
-from logs import lg
-
-from lib import packetid
-from lib import strng
+    sys.path.append(_p.join(_p.dirname(_p.abspath(sys.argv[0])), ".."))
 
-from userid import my_id
-from userid import global_id
+# ------------------------------------------------------------------------------
 
-from system import nonblocking
-from system import tmpfile
-
-from main import settings
-from main import events
+from crypt import encrypted, key
 
 from automats import automat
+from lib import packetid, strng
```

Comment:
I was trying to find a way in `isort` to disable grouping of the imports, but was not able to find the right flag.

Basically I prefer to have 
```
from lib import packetid
from lib import strng
```

instead of:
```
from lib import packetid, strng
```

It is probably not a common coding style, but there are some reasons for that...

If it is too complicated, can we at the moment skip applying `isort` in the pre-commit? I would like to take my time and find a solution for that...